docker-compose.yml

```diff
@@ -101,6 +101,7 @@ services:
   api:
     depends_on:
       - elasticsearch
+      - elasticsearch_test
```

Comment:
Will we have a test elasticsearch container running in production?

Reply:
No, this repository is for developers only.

Reply:
GitHub isn't letting me respond to the other questions of
> So, how do we run the tests now?

But here's the answer:

```
fig up -d
fig exec api carton exec -- prove -lvr t/
```

start up the containers and then run the tests in the api container. There are some test failures that will need investigation. The containers do start and the tests do populate the elasticsearch_test instance.